torch/csrc/distributed/rpc/init.cpp

```diff
@@ -9,7 +9,9 @@
 #include <torch/csrc/distributed/rpc/request_callback_impl.h>
 #include <torch/csrc/distributed/rpc/rpc_agent.h>
 #include <torch/csrc/distributed/rpc/rref_context.h>
+#ifndef _WIN32
 #include <torch/csrc/distributed/rpc/tensorpipe_agent.h>
+#endif
```

Comment:
This should not be necessary after #42682 (which is landing right now).

Reply:
And the same applies to every other spot in this PR where you add a `_WIN32` guard around TensorPipe.